compat/src/render.js

```diff
@@ -142,29 +148,29 @@ options.vnode = vnode => {
 				// value will be used as the file name and the file will be called
 				// "true" upon downloading it.
 				value = '';
-			} else if (/ondoubleclick/i.test(i)) {
+			} else if (DOUBLE_CLICK.test(lowerCased)) {
 				i = 'ondblclick';
 			} else if (
-				/^onchange(textarea|input)/i.test(i + type) &&
+				CHANGE_INPUT.test(lowerCased + type) &&
 				!onChangeInputType(props.type)
 			) {
-				i = 'oninput';
-			} else if (/^onfocus$/i.test(i)) {
+				lowerCased = i = 'oninput';
+			} else if (ON_FOCUS.test(lowerCased)) {
 				i = 'onfocusin';
-			} else if (/^onblur$/i.test(i)) {
+			} else if (ON_BLUR.test(lowerCased)) {
 				i = 'onfocusout';
-			} else if (/^on(Ani|Tra|Tou|BeforeInp|Compo)/.test(i)) {
+			} else if (ON_ANI.test(i)) {
 				i = i.toLowerCase();
```

Comment:
Could this be replaced with `lowerCased`?

```suggestion
				i = lowerCased;
```